src/app/shared/store/meta-data.state.ts

```
@@ -169,10 +175,22 @@ export class MetaDataState {
     return state.codeficator;
   }
 
+  @Selector()
+  static languageList(state: MetaDataStateModel): LanguageListItem[] {
+    return state.languageList;
+  }
+
   @Action(GetDirections)
   getDirections({ patchState }: StateContext<MetaDataStateModel>, {}: GetDirections): Observable<Direction[]> {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)